source/api/events/catalog-of-events.md

```diff
@@ -181,6 +181,45 @@ Cypress.on('uncaught:exception', (err, runnable) => {
 })
 ```
 
+### To conditionally turn off uncaught exception handling for a certain error
+
+```javascript
+// likely want to do this in a support file
+// so it's applied to all spec files
+// cypress/support/index.js
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+  // we expect a 3rd party library to error and don't want
+  // it to fail the test so we return false
+  if (err.message.includes('3rd party error')) {
+    return false
+  }
```

Comment:
I kind of feel like people will interpret this as a way to turn off all 3rd party error catching. 😅 

```suggestion
  // we expect a 3rd party library error with message 'list not defined'
  // and don't want to fail the test so we return false
  if (err.message.includes('list not defined')) {
    return false
  }
```